assets/js/translator.js

```diff
@@ -460,6 +460,11 @@ function populateTranslationList() {
         numDstCols = Math.ceil(dstLangs.length / TRANSLATION_LIST_ROWS) < (TRANSLATION_LIST_COLUMNS + 1)
             ? Math.ceil(dstLangs.length / TRANSLATION_LIST_ROWS)
             : TRANSLATION_LIST_COLUMNS;
+
+    var thresholdWindowWidth = 1000;
```

Comment:
Ohh Um! yeah, I guess I was doing a trial and error and arrived at this number which will dynamically accommodate everything! I shall get a more concrete number.